### PR TITLE
Changes the mateba sprites, deletes unused mateba sprites. changes some stats on the ammo and gun.

### PR DESCRIPTION
**Pull request by Pariah919**
## About The Pull Request
Gives the mateba a new sprite and deletes some unused mateba sprites, deletes the admiral mateba and remakes it into a not marine mateba.
Captain mateba is now just a mateba with different description.
Mateba no longer has a damage multi, gains some attachments, does 50/20 and nerfs some of the stagger/slowdown it does, it no longer has recoil when you're two handing it.

## Why It's Good For The Game
Makes the mateba visually consistent with the game in general, makes it actually useable as a gun without the annoying screenshake. Cleans up some unused sprites.

## Changelog
:cl:
add: Added new mateba sprite.
del: Removed old mateba sprites that are unused.
tweak: tweaked the admiral mateba into a non-marine mateba. It was only used by DS.
balance: Mateba has a few new attachment options, no longer has recoil when 2-handed. Now deals 50/20 instead of 47.5/15. Reduced stagger/slowdown.
/:cl:

### CODE REVIEW

**Comment by tgstation-server** [comment]
You currently have a negative Fix/Feature pull request delta of -20. Maintainers may close this PR at will. Fixing issues or improving the codebase will improve this score.

**Comment by Pariah919** [comment]
>You currently have a negative Fix/Feature pull request delta of -20. Maintainers may close this PR at will. Fixing issues or improving the codebase will improve this score.
>
I've done the impossible

File: code/modules/projectiles/guns/revolvers.dm [hunk 1]
```diff
@@ -422,28 +422,29 @@
...
-	icon_state = "a_mateba"
-	item_state = "a_mateba"
 
 /obj/item/weapon/gun/revolver/mateba/captain
-	name = "\improper Mateba autorevolver special"
-	desc = "The Mateba is a powerful, fast-firing revolver that uses its own recoil to rotate the cylinders. It uses heavy .454 rounds. This version is a limited edition produced for the TGMC, and issued in extremely small amounts."
-	icon_state = "c_mateba"
-	item_state = "c_mateba"
+	name = "\improper TL-24 autorevolver special"
+	desc = "The Mateba is a powerful, fast-firing revolver that uses its own recoil to rotate the cylinders. This one appears to have had more love and care put into it, Uses .454 rounds."
```
Comment (XSlayer300) [inline code/modules/projectiles/guns/revolvers.dm:445]: Lowercase for `Uses .454 rounds.`

Comment (Pariah919) [reply code/modules/projectiles/guns/revolvers.dm:445]: oh I put a comma

**Comment by tgstation-server** [comment]
Your Fix/Feature pull request delta is currently below zero (-20). Maintainers may close future Feature/Tweak/Balance PRs. Fixing issues or helping to improve the codebase will raise this score.

**Pull request merged**